haskell/src/Ligo/BaseDAO/TZIP16Metadata.hs

```diff
@@ -57,12 +70,27 @@ mkMetadataSettings msConfig = MetadataSettings
 -- without user's input.
 knownBaseDAOMetadata :: MetadataSettings -> Metadata (ToT Storage)
 knownBaseDAOMetadata settings = mconcat
-  [ version . fromString $ showVersion Paths.version
+  [ name "Base DAO"
+  , description "A generic smart contract on Tezos that enables a community to collectively govern resources, registries, or rules."
+  , version . fromString $ showVersion Paths.version
   , license $ License "MIT" Nothing
   , authors [Author "Serokell", Author "Tocqueville Group"]
   , homepage "https://github.com/tezos-commons/baseDAO"
-  , interfaces [tzip 17]
```

Comment:
:+1:

Reply:
Hmm, I replace this with tzip-16 by mistake. Should I add it back? cc @pasqu4le

Reply:
I noticed this and though it was intentional.

I checked and it seemed that indeed we don't properly support TZIP-17 (at least not the current spec), but rather a mechanism similar to it (please correct me if I'm wrong), so I guess it's ok to remove this.